### PR TITLE
Music maker

### PR DESCRIPTION
**Pull request by SurajNarang**

### CODE REVIEW

File: .vscode/settings.json [hunk 1]
```diff
@@ -0,0 +1,3 @@
+{
```
Comment (tecanal) [inline .vscode/settings.json:1]: This file shouldn't be committed.

File: play/symphony-conductor/index.html [hunk 2]
```diff
@@ -1,130 +1,243 @@
...
+                    </textarea>
+
+                <button onclick="myFunction(`myInput0`)" id="exampleButton0">Copy (Beginner Level)</button>
+
+                <button onclick="myFunction(`myInput`)" id="exampleButton1">Copy (Intermediate Level)</button>
+
+                <button onclick="myFunction(`myInput2`)" id="exampleButton2">Copy (Complex Level)</button>
+
+                <video width="329" height="190" controls id="tutorialVideo">
+                    <source src="TutorialVideo.mp4" type="video/mp4">
```
Comment (tecanal) [inline play/symphony-conductor/index.html]: I uploaded this file to YouTube in order to not have to commit such a large video file. Could you change this to the YouTube URL: https://youtu.be/0kg-ZQXIpFY

File: play/symphony-conductor/js/app.js [hunk 3]
```diff
@@ -122,6 +140,15 @@ const INSTRUMENTS = [
         code: `play("Gong");`,
         link: "https://en.wikipedia.org/wiki/Gong"
         
+    },
+    {
+        name: 'Hand-Clap',
+        description: "Hand-Clap: Considered a percussion instrument",
+        img: 'img/handclap.png',
+        sound: 'sounds/handclap.wav',
+        code: `play("Hand-Clap");`,
```
Comment (tecanal) [inline play/symphony-conductor/js/app.js]: Could you remove the capitalization in the code to match the rest of the instruments?

**Review by tecanal — CHANGES_REQUESTED** [review]
Change the video link and remove the video because committing large binary files like videos to Git is not usually a good idea

File: play/symphony-conductor/index.html (hunk 2 above)
Comment (SurajNarang) [reply play/symphony-conductor/index.html]: I switched the video src from the mp4 file to the youtube link

File: .vscode/settings.json (hunk 1 above)
Comment (SurajNarang) [reply .vscode/settings.json:1]: I will make note of this for future commits

**Pull request merged**